packages/babel-plugin-proposal-decorators/test/fixtures/2021-12-classes/expressions-named-evaluation/exec.js

```diff
@@ -0,0 +1,158 @@
+// prettier-ignore
+const logs = [];
+const decFactory = (logs) => (value, context) => {
+  expect(value.name).toEqual(context.name);
```

Comment:
The test is more hardened than #15122, here we further assert `target.name` should equal `context.name`.